smartling-job-batches-api/src/main/java/com/smartling/api/jobbatches/v2/pto/CreateJobResponsePTO.java

```diff
@@ -0,0 +1,24 @@
+package com.smartling.api.jobbatches.v2.pto;
+
+import com.smartling.api.v2.response.ResponseData;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class CreateJobResponsePTO implements ResponseData
+{
+    private String translationJobUid;
+    private String jobName;
+    private String jobNumber;
+    private String dueDate;
```

Comment:
I don't see `dueDate` in the response. I think it makes sense to accept it too (as optional).

Reply:
These fields are exaclty the same as defined in the endpoint response, including `dueDate`.